### PR TITLE
OSDOCS-2461: Created a new book covering the OpenShift Cluster Manager.

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR creates a new book covering the OCM tool. Copy of #38655. 

Versions: 4.9+

JIRA: https://issues.redhat.com/browse/OSDOCS-2461

Preview: https://deploy-preview-39178--osdocs.netlify.app/openshift-dedicated/latest/ocm/ocm-overview

### CODE REVIEW

**Comment by EricPonvelle** [comment]
@cblecker - I broke my last PR, but I made your suggested changes. Anything else that you see that needs to be changed?

Do you know who would QE this?

**Comment by gregsheremeta** [comment]
> Do you know who would QE this?

probably me, but I need some help. The Jira is lacking some context. Is any of this new content, or are we simply linking existing stuff from the existing OCM docs into the OSD docs?

**Comment by EricPonvelle** [comment]
> > Do you know who would QE this?
> 
> probably me, but I need some help. The Jira is lacking some context. Is any of this new content, or are we simply linking existing stuff from the existing OCM docs into the OSD docs?

@gregsheremeta, so basically, there's nothing in our documentation around OCM, so the ask is vague because there's just nothing. @okashi18 went the OCM with me and described all of the tabs and possible actions within OCM.

So, I guess if you can verify all of my descriptions of everything and if we are missing more information, we can highlight that for future updates. Does that clarify or complicate things?

**Comment by gregsheremeta** [comment]
It clarifies, but now you have me worried we're doing duplicate work. The existing OCM docs are here:
https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021-02/html/managing_clusters/assembly-what-is-ocm

So if the goal is to have OCM docs under the OSD stuff, we should just copy what has already been written, right?

**Comment by gregsheremeta** [comment]
> The existing OCM docs at https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021-02/html/managing_clusters/assembly-what-is-ocm
> how accurate/up-to-date [are those]?

I'm not sure. I'm not familiar with who wrote it and how much it was kept up to date. @wgordon17 or @luciddreamz do you know?

File: ocm/ocm-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+include::modules/ocm-accessing.adoc[leveloffset=+1]
+
+[id="ocm-general-actions"]
+== General actions
+
+On the top right of the cluster page, there are some actions that a user can perform on the entire cluster.
+
+. **Open console** launches a web console so that the cluster owner can issue commands to the cluster.
+. **Actions** dropdown menu allows the cluster owner to rename the display name of the cluster, change the amount of load balancers and persistent storage on the cluster, if applicable, manually set the node count, and delete the cluster.
```
Comment (xueli181114) [inline ocm/ocm-overview.adoc:17]: There is another action in the dropdown with capability set, like "Hibernate cluster". Should it be mentioned here?

File: ocm/ocm-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+. **Refresh** icon forces a refresh of the cluster.
+
+[id="ocm-cluster-tabs"]
+== Cluster Tabs
+
+Selecting an active, installed cluster shows tabs associated with that cluster. The following tabs display once the cluster's installation completes:
```
Comment (xueli181114) [inline ocm/ocm-overview.adoc:23]: Should we clarify that the below tabs only show for `Managed` OSD clusters?

File: modules/ocm-overview-tab.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+1. **Availability** shows which type of availability zone that the cluster uses, either single or multizone.
+1. **Version** is the OpenShift version that is installed on the cluster. If there is an update available, you can update from this field.
+1. **Created at** shows the date and time that the cluster was created.
+1. **Owner** identifies who created the cluster and has owner rights.
+1. **Subscription type** shows the subscription model that was selected on creation.
+1. **Infrastructure type** is the type of account that the cluster uses.
+1. **Status** displays the current status of the cluster.
+1. **Total vCPU** shows the total available virtual CPU for this cluster.
+1. **Total memory** shows the total available memory for this cluster.
+1. **Load balancers**
```
Comment (xueli181114) [inline modules/ocm-overview-tab.adoc:23]: No introduction for `Load balancers` ?

File: modules/ocm-accesscontrol-tab.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+[id="ocm-accesscontrol-tab_{context}"]
+= Access control tab
+
+This tab allows the cluster owner to grant roles to other users.
+
+.Prerequisites
+
+. You must be the cluster owner or have the correct permissions to grant roles on the cluster.
+
+.Procedure
```
Comment (xueli181114) [inline modules/ocm-accesscontrol-tab.adoc:14]: This tab is not only for role grant in oCM. It also support IDP configuration and aws infrastructure access grant. Is it enough to just introduce the OCM role grant?

File: modules/ocm-networking-tab.adoc [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+// ocm/ocm-overview.adoc
+
+[id="ocm-networking-tab_{context}"]
+= Networking tab
+
+The networking tab provides a control plane API endpoint as well as the default application router. Both the control plane API endpoint and the default application router may be made private by selecting the respective box below each of them.
+
+[NOTE]
+====
+For STS installations, these options cannot be changed.
```
Comment (xueli181114) [inline modules/ocm-networking-tab.adoc:12]: This tab is not supported for standard GCP clusters without capability "capability.organization.create_gcp_non_ccs_cluster" . Should we clarify it here?

Comment (wgordon17) [reply modules/ocm-networking-tab.adoc:12]: STS should _currently_ only show up for ROSA clusters, not even for OSD clusters yet. And ROSA is AWS-only, so it should be unnecessary to call out lack of GCP support

File: modules/ocm-overview-tab.adoc (hunk 3 above)
Comment (EricPonvelle) [reply modules/ocm-overview-tab.adoc:23]: @xueli181114 and @gregsheremeta I really didn't know what would go here? Any ideas?

File: ocm/ocm-overview.adoc (hunk 1 above)
Comment (EricPonvelle) [reply ocm/ocm-overview.adoc:17]: @xueli181114, sure thing! What does the hibernate cluster action do specifically? Does it just pause the cluster until a user needs it? Any other considerations to add?

File: modules/ocm-networking-tab.adoc (hunk 5 above)
Comment (xueli181114) [reply modules/ocm-networking-tab.adoc:12]: This comment is not related to sts. I mean the networking tab is not supported for standard GCP clusters expect the organization with "capability.organization.create_gcp_non_ccs_cluster"

File: ocm/ocm-overview.adoc (hunk 1 above)
Comment (xueli181114) [reply ocm/ocm-overview.adoc:17]: @EricPonvelle  I agree with @wgordon17 's comments. Let's skip it.

File: ocm/ocm-overview.adoc [hunk 6]
```diff
@@ -0,0 +1,44 @@
+[id="ocm-overview"]
+= Red Hat OpenShift Cluster Manager (OCM)
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: ocm-overview
+toc::[]
+
+The Red Hat OpenShift Cluster Manager (OCM) allows users to create, manage, and delete OpenShift clusters using a web UI.
```
Comment (0kashi) [inline ocm/ocm-overview.adoc:7]: Can we add a little bit more introductory context here? Something like (feel free to wordsmith):

_OpenShift Cluster Manager is a managed service where you can install, modify, operate and upgrade your Red Hat OpenShift clusters. It allows you to work with all of your organization’s clusters from single dashboard.  OpenShift Cluster Manager provides the links and steps to install OpenShift Container Platform, ROSA and OSD clusters and is responsible for managing both OpenShift Container Platform clusters after self-installation and your ROSA and OSD clusters.

From OpenShift Cluster Manager, you can:
- Create new clusters
- View cluster details and metrics
- Manage your clusters (scale, node labels, networking, authentication, etc)
- Manage access control
- Monitor clusters
- Schedule upgrades_

(Above modified from here: https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html/managing_clusters/assembly-what-is-ocm)

**Comment by EricPonvelle** [comment]
@0kashi, I updated the PR with your text suggestions. I think it really provides a lot of context for OCM. Let me know what you think.

**Comment by 0kashi** [comment]
The intro looks good.  

On the Access Control tab, let's add that it also allows granting elevated permissions, and setting up an Identity Provider.  

Can we also modify "Settings Tab" > "Update Strategy" to "...or if all updates are ~~done~~ scheduled manually"?

**Comment by EricPonvelle** [comment]
@0kashi updated!

**Comment by 0kashi** [comment]
lgtm

**Pull request merged**